_rules/SC4-1-2-aria-attr-valid.md

```diff
@@ -0,0 +1,114 @@
+---
+name: Validity of ARIA attribute
+
+description: |
+ This rule checks that the `aria-` attribute specified is defined in ARIA 1.1.
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+Any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) with an attribute that starts with `aria-`.
+
+### Expectation
+
+Each target element that has an attribute which starts with `aria-` is defined in ARIA 1.1.
+
+## Assumptions
+
+*There are currently no assumptions*
```

Comment:
This has a basic assumption that `aria-` attributes are only used to express accessibility information that can be picked up by assistive technologies. If someone decides to make up their own `aria-` attribute and do some scripting to do other stuff with it, this rule won't account for that.